bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -32,6 +32,7 @@ tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
 tokio-stream = "0.1.8"
 uniffi = { workspace = true }
 uniffi_macros = { workspace = true }
+zeroize = { workspace = true, features = ["zeroize_derive"] }
```

Comment:
Since you didn't use the derive macro, you don't need this feature either.

Reply:
Thanks for fixing this 🙌